solana/bridge_token_factory/programs/bridge_token_factory/build.rs

```diff
@@ -4,7 +4,7 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
 
     let program_id =
-        env::var("PROGRAM_ID").unwrap_or("dahPEoZGXfyV58JqqH85okdHmpN8U2q8owgPUXSCPxe".into());
+        env::var("PROGRAM_ID").unwrap();
```

Comment:
@diegofigs we want to remove the default  `PROGRAM_ID` to avoid mistakes during the building and deployment but we break the clippy. Is it possible to set this env variable for the backstage flow, eg adding it to the `.catalog-info.yam`? 
CC @kiseln

Reply:
@karim-en there's a workaround: the workflow calls a Makefile's `env` script, in there you could define the `PROGRAM_ID`